packages/client-generator-ts/src/generateClient.ts

```diff
@@ -213,10 +213,6 @@ export async function generateClient(options: GenerateClientOptions): Promise<vo
 
   const clientEngineType = getClientEngineType(generator)
```

Comment:
It doesn't seem like we are using `clientEngineType` in this scope after removing  the queryCompiler check, are we? 🤔
```suggestion
```

Reply:
Sorry, my bad, don't know how I missed that. Thanks for pointing it out!